workbench-core/swb-common-ui/src/context/NotificationContext.tsx

```diff
@@ -16,14 +16,14 @@ export interface NotificationProps {
   closeNotification: (id: string) => void;
 }
 
-const NotificationsContext: Context<NotificationProps> = createContext({
+const NotificationsContext: React.Context<NotificationProps> = React.createContext({
   notifications: {} as Notifications,
   displayNotification: (id: string, notification: FlashbarProps.MessageDefinition) => {},
   closeNotification: (id: string) => {}
 });
 
 export function NotificationsProvider({ children }: { children: React.ReactNode }): JSX.Element {
-  const [allNotifications, setAllNotifications] = useState<Notifications>({});
+  const [allNotifications, setAllNotifications] = React.useState<Notifications>({});
```

Comment:
Does just calling `useState` itself cause an eslint error? Why do we need to use `React` now?

Reply:
Yeah, `useState` is not recognized in this package (not sure why, we have all the same react-specific packages listed as `swb-ui`). Which is why had to specify it's a part of React here.

Reply:
is there any overlap in function names with react and our code?